src/main/resources/db/migration/V4_3__standardtekster_init.sql

```diff
@@ -0,0 +1,26 @@
+INSERT INTO STANDARDTEKST VALUES (DEFAULT, now(), now(), 'INNKALLING_SM_OVERSKRIFT', '1900-01-01T00:00:00', 'Til den sykmeldte');
+INSERT INTO STANDARDTEKST VALUES (DEFAULT, now(), now(), 'INNKALLING_SM_INNLEDNING_OVERSKRIFT', '1900-01-01T00:00:00', 'Innkalling til dialogmøte');
```

Comment:
Ser nå at det blir veldig mange tekster. Burde vi ikke ha en kolonne som reflekteret type dokument/varsel?
Slik det er nå må vi søke på prefix-string.

Om vi har en kolonne som reflekter som heter f.eks type(varselTyp)  og henter den fra f.eks:
`enum class MotedeltakerVarselType {
    AVLYST,
    INNKALT,
    NYTT_TID_STED,
}`

Reply:
Apropos mange tekster: Hvor blir det bestemt hva som skal være versjonerte standardtekster? Må vi har versjon på f.eks. `Med hilsen` og `Møtested`, bryter vi loven hvis vi endrer fra `Møtested` til `Sted for møte` i en eksisterende innkalling? Og hvis man endrer fra `Møtested` til `Veldig lang tekst som ikke ligner på "Møtested" i det hele tatt`, burde det kanskje uansett bli en ny nøkkel fordi det egentlig er noe helt annet?

Reply:
Enig i at det kan være nyttig med enum-kolonner for varseltype og deltakertype. Lettere å spørre på og mer tydelig hva standardteksten gjelder (synes ikke SM/AT/NL-forkortelsene var helt åpenbare 😅 )